<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CChannelAttribute</class>
 <widget class="QDialog" name="CChannelAttribute">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改控制器属性</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="btnOk">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>11</y>
     <width>215</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>控制器名称：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>控制器地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spAddr">
        <property name="maximum">
         <number>63</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>PC机IP地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtIP">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>IPC机IP地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtIPCIP">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
